job_runner/reporting/file.py

```diff
@@ -142,36 +135,35 @@ def _add_log_line(self, job):
     # This file will be read by the job_queue_node_fscanf_EXIT() function
     # in job_queue.c. Be very careful with changes in output format.
     def _dump_error_file(self, job, error_msg):
-        fileH = open(self.ERROR_file, "a")
-        now = time.localtime()
-        fileH.write("<error>\n")
-        fileH.write(
-            "  <time>{:02d}:{:02d}:{:02d}</time>\n".format(
-                now.tm_hour, now.tm_min, now.tm_sec
+        with open(self.ERROR_file, "a") as fileH:
+            now = time.localtime()
+            fileH.write("<error>\n")
```

Comment:
While you are refactoring might as well change variable name

Reply:
Why is it called "fileH"? is it something to do with "heading"?

Reply:
No clue, suspect `file_handle`, but you are free to name it something more sensible.